Gomfile

```diff
@@ -1,3 +1,4 @@
+gom 'github.com/stripe/stripe-go', :fork => 'github.com/dicefm/stripe-go', :branch => 'dynamic-last-four'
```

Comment:
Also need to work out how we bring our changes into the upstream, we've diverged a bit atm.